src/components/MultiSig/components/RecipientsPage/RecipientsPage.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react';
+import { AddRecipientPageProps, RecipientAmount } from './types';
+import styles from '../../MultiSig.module.sass';
+import { isValidSeiAddress } from '@sei-js/core';
+import { toast } from 'react-toastify';
+import TableWithDelete from '../Utils/TableWithDelete';
+import { BiSpreadsheet } from '@react-icons/all-files/bi/BiSpreadsheet';
+import CSVUpload from './CSVUpload';
+import Drawer from 'react-modern-drawer'
+import cn from 'classnames';
+
+const RecipientsPage = ({multiSigAccount, handleBack, setFinalizedRecipients, setParsedRecipients, parsedRecipients, setTxMemo, txMemo}: AddRecipientPageProps) => {
+    const [isPaneOpen, setIsPaneOpen] = useState<boolean>(false);
+	const [recipientAddress, setRecipientAddress] = useState<string>('');
+	const [recipientAmount, setRecipientAmount] = useState<number>(0);
+    const [recipientDenom, setRecipientDenom] = useState<string>('usei');
+    
+    const renderRecipientsPage = () => {
+        const renderRecipientList = () => {
+            if (parsedRecipients.length === 0) return null;
+
+            return (
+                <TableWithDelete
+                    items={parsedRecipients}
+                    setItems={setParsedRecipients}>
+                </TableWithDelete>
+            );
+        };
+
+        const renderRecipientContent = () => {
+            if (parsedRecipients.length !== 0) return null;
+
+            return (
+                <>
+                    <div className={styles.cardTip}>
+                        <BiSpreadsheet className={styles.tipBulb} />
+                        <div>
+                        <p>Upload a CSV file with two columns "Recipient" and "Amount" for all the addresses you would like to send funds to</p>
+                        <br/>
```

Comment:
nit: instead of using `<br />`, it's generally cleaner to just add style to the paragraph below/above to add margin